bats/fb-katello-proxy.bats

```diff
@@ -30,6 +30,36 @@ setup() {
   hammer capsule content synchronize --id=$PROXY_ID
 }
 
+@test "setup repositories and publish content view version for auto-proxy-sync test" {
+  hammer repository create --organization="${ORGANIZATION}" \
+    --product="${PRODUCT}" --content-type="yum" --name "${YUM_REPOSITORY_5}" \
+    --url https://fixtures.pulpproject.org/rpm-complex-pkg/ | grep -q "Repository created"
+  hammer repository synchronize --organization="${ORGANIZATION}" \
+    --product="${PRODUCT}" --name="${YUM_REPOSITORY_5}"
+  repo_id=$(hammer --csv --no-headers repository list --organization="${ORGANIZATION}" \
+    | grep "${YUM_REPOSITORY_5}" | cut -d, -f1)
+  hammer content-view add-repository --organization="${ORGANIZATION}" \
+    --name="${CONTENT_VIEW}" --repository-id=$repo_id | grep -q "The repository has been associated"
+  hammer content-view publish --organization="${ORGANIZATION}" --name="${CONTENT_VIEW}"
+}
+
+@test "auto-synced content is available from proxy in the Library LCE using /pulp/content" {
+  local next_wait_time=0
+
+  until tCheckPulpYumContent "${PROXY_HOSTNAME}" "pulp/repos" "Library" "complex-package-2.3.4-5.el8.x86_64.rpm" "${YUM_REPOSITORY_5_LABEL}"; do
+    if [ $next_wait_time -eq 14 ]; then
+      tCheckPulpYumContent "${PROXY_HOSTNAME}" "pulp/repos" "Library" "complex-package-2.3.4-5.el8.x86_64.rpm" "${YUM_REPOSITORY_5_LABEL}"
+    fi
+    sleep $(( next_wait_time++ ))
+  done
+}
+
+@test "sync just one content view to the proxy" {
+  tSkipIfOlderThan43
+
+  hammer capsule content synchronize --id=$PROXY_ID --content-view="${CONTENT_VIEW}" --organization="${ORGANIZATION}"
+}
+
```

Comment:
given the restructure, this now tests that you could issue a sync command for a CV, but never checks "back" if the content was actually received (as the result of that command, the content is already on the proxy due to the previous steps)?

Reply:
Yeah, the previous auto-sync from the cvv publish is what we really wanted to test, not the sync with a `--content-view` argument. I realized that after talking with Justin. I'm just testing here that the cv-restricted sync is run without error.  I could've removed this artifact from my prior assumption, but it's checking that the API works at least.

Reply:
Let me know if anyone thinks it'd be worth syncing different content to test this.  The content is already there, so, as-is, it's not worth checking if the RPM exists again.

Reply:
I'm cool with this "as is". Was just checking that I understand the tests correctly.

Reply:
Would you mind adjusting the commit message to point out that this adds tests for two things?
1. proxy-auto-sync (or whatever the correct term is) for all Katello versions
2. Sync specific CV for 4.3+?